index.html

```diff
@@ -0,0 +1,184 @@
+        </div>
+        <nav>
+            <img class="hamburger" id="hamburger" width="40px" src="images/588a64e7d06f6719692a2d11.png" alt="">
+            <div class="hamburger-container">
+                <div class="close"><img width="30px" id="closeBtn"
+                        src="images/67-679412_x-cross-close-symbol-icon-button-gui-grey.png" alt=""></div>
+
+                <ul class="hamburger-items ">
+                    <li><a href="about.html" class="items" id="menuabout">About</a></li>
+                    <li><a href="" class="items" id="program">Program</a> </li>
+                    <li><a href="" class="items" id="join">Join</a></li>
+                    <li><a href="" class="items" id="Sponsor"> Sponsor</a></li>
+                    <li><a href="" class="items" id="enlist">Enlist</a> </li>
+                </ul>
+
+            </div>
+            <ul class="nav-items">
+                <li><a href="about.html" class="item" id="menuabout">About</a></li>
+                <li><a href="" class="item" id="program">Program</a> </li>
+                <li><a href="" class="item" id="join">Join</a></li>
+                <li><a href="" class="item" id="Sponsor"> Sponsor</a></li>
+                <li><a href="" class="item" id="enlist">Enlist</a> </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="heading">
+            <div class="header-content">
+                <div class="heading-content">
+                    <div class="hello">Hello Music world!</div>
+                </div>
+                <h1 class="main-header">The Global Musical Concert 2022</h1>
+
+                <div class="heading-content2">
+                    <p class="supporting-text">A joyful celeberation believing in the value of openess and sharing,
+                        with musician from over 80 countries is taking place in october,in London Uk
+                    </p>
+                </div>
+                <div class="date">2022.10.15(Thur)</div>
+                <div class="date-support">@ National theatre of arts, London</div>
+            </div>
+
+        </section>
+        <section class="main-program">
+            <h2>Main program</h2>
+            <div class="main-program-container">
+                <div class="box">
+                    <img width="70px" height="100px"
+                        src="images/kissclipart-lecturer-icon-clipart-lecturer-computer-icons-55c69a33907f2b16.png"
+                        alt="">
+                    <h4 class="box-main">Lectures</h4>
+                    <div class="box-text">
+                        Get adressed by some of the best musician from all around the world
+                    </div>
+                </div>
+                <div class="box">
+                    <img width="70px" height="100px" src="images/exhibition.png" alt="">
```

Comment:
- [ ] if the image has a primary role in the content you need to write the `alt` attribute and not leave it blank so please try to write in all images that have primary in the content of you page